Dockerfile

```diff
@@ -1,26 +1,17 @@
-FROM debian:sid
+FROM openjdk:8-jre-alpine
 
 MAINTAINER Christian Kuntzsch <christian@kuntzsch.me>
 
-RUN \
-    apt-get update && \
-    apt-get install -y openjdk-8-jre wget
-
-ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64
```

Comment:
If you use the openjdk image you don't have to set these environment variables...